test/integration/apiserver/cel/validatingadmissionpolicy_test.go

```diff
@@ -283,22 +297,6 @@ func Test_ValidateNamespace_NoParams(t *testing.T) {
 	}
 	for _, testcase := range testcases {
 		t.Run(testcase.name, func(t *testing.T) {
-			featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, genericfeatures.ValidatingAdmissionPolicy, true)
-
-			server, err := apiservertesting.StartTestServer(t, nil, []string{
-				"--enable-admission-plugins", "ValidatingAdmissionPolicy",
-			}, framework.SharedEtcd())
-			if err != nil {
-				t.Fatal(err)
-			}
-			defer server.TearDownFn()
-
-			config := server.ClientConfig
-
-			client, err := clientset.NewForConfig(config)
-			if err != nil {
-				t.Fatal(err)
-			}
```

Comment:
you have to clean up resources on this test in each iteration, it fails because of

> > b0ec-3ea2aecb1609" srcIP="127.0.0.1:51970" apf_pl="exempt" apf_fs="exempt" apf_iseats=1 apf_fseats=0 apf_additionalLatency="0s" apf_execution_time="2.572543ms" resp=409
>     validatingadmissionpolicy_test.go:302: validatingadmissionpolicies.admissionregistration.k8s.io "validate-namespace-suffix" already exists
>     --- FAIL: Test_ValidateNamespace_NoParams/runtime_error_when_validating_namespace,_but_failurePolicy=Fail (0.00s)

Reply:
it seems this test is missing the

```
		if err := cleanupPolicy(t, client, policy, testcase.policyBinding); err != nil {
				t.Fatalf("error while cleaning up policy and its bindings: %v", err)
			}
```

Reply:
> you have to clean up resources on this test in each iteration,
yeah I did miss the cleanupPolicy() here, but tweaking the refresh-policy interval also helped.